docs/guides/sortable-image-tags.md

```diff
@@ -0,0 +1,150 @@
+<!-- -*- fill-column: 100 -*- -->
+# How to make sortable image tags to use with automation
+
+Flux v2 does not support selecting the lastest image by build time. Obtaining the build time needs
+the container config for each image, and fetching that is subject to strict rate limiting by image
+registries (e.g., by [DockerHub][dockerhub-rates]).
+
+This guide explains how to construct image tags so that the most recent image has the tag that comes
+last in alphabetical order. The technique suggested is to put a timestamp or serial number in each
+image tag.
+
+## Formats and alternatives
+
+The important properties for sorting alphabetically are that the parts of the timestamp go from most
+significant to least (e.g., the year down to the second), and that the output is always the same
+number of characters.
+
+Image tags are often shown in user interfaces, so readability matters. Here are some alternatives:
+
+```bash
+$ # seconds-since-epoch (used in the example above)
+$ date +%s
+1611840548
+$ # date and time (remember ':' is not allowed in a tag)
+$ date +%F.%H%M%S
+2021-01-28.133158
+```
+
+Alternatively, you can use a stable serial number as part of the tag.  Some CI platforms will
+provide a build number in an environment variable, but that may not be reliable to use as a serial
+number -- check the platform documentation.
+
+A commit count can be a reasonable stand-in for a serial number, if you build an image per commit,
```

Comment:
I don't think this will work with alphabetic ordering, you need to use semver e.g. `1.0.$REV_COUNT`. I would remove this from the docs as it's error prone.

Reply:
Hmm yes, you would need to pad it to a particular number of characters, for it to sort correctly.

Reply:
Easier to remove it than to make it workable.